### ISSUE TITLE
Initial state of menu when using sidebar addon

### ISSUE THREAD

**Issue by jackson-dikey** [FrDH/mmenu-js#967]
Greetings. 
Thank you for such a great job and contribution to open source.
I have a question. 
Is it possible to somehow set the initial state of the menu in state of **closed** when using sidebar addon? At the moment the menu is always **open** when using this addon. And if there is no such possibility at the moment., it would be very cool to add this feature to the settings.
Thank you in advance! :)

**Comment by FrDH** [comment]
I guess you could invoke the "close" method in the API directly after creating the menu, but I'm not sure if that would use an animation or not:

````js
var API = $("#my-menu").mmenu({
   // the options
}).data( "mmenu" );

API.close();
````

I'll see if I can introduce an option in the sidebar add-on for better support.

**Comment by jackson-dikey** [comment]
This is the first that I tried to apply. Animation occurs and it is visible when the page is loaded. It is necessary to initially closed state when loading the page. I think this option will be useful to many. If possible, let us know in this issue when it will be possible to realize this opportunity. Thanks again! :)

**Comment by FrDH** [comment]
I'll add the "next release" tag as soon as iI got it to work.

**Comment by FrDH** [comment]
I got something to work, where you can set `expanded.initial` to be either `open` (the default), `closed` or `remember` in [the develop branche](https://github.com/FrDH/mmenu-js/tree/develop):
- `open` : behaves as it did before: opens the sidebar based on the `use` option.
- `closed`: does not expand the sidebar untill the user desides to open the menu.
- `remember`: expands the sidebar if it was expanded in the previous pageLoad (using localStorage).

If you will, please give it a go and let me know what you think.

**Comment by FrDH** [comment]
should be fixed in 8.2
